pom.xml

```diff
@@ -66,6 +66,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+            <scope>${dependency.scope}</scope>
```

Comment:
Why does guava need to be added, doesn't Hadoop have this already?

Reply:
> Why does guava need to be added, doesn't Hadoop have this already?

Yes it should do, but for some reason, wrong version of Guava is used for unit test and it causes NoSuchMethodError. Some other libraries that have a transitive dependency on Guava seem to cause this problem. For resolving it, I'll change the dependency declaration order in pom.xml, instead of adding this dependency.